locale/translators/index.php

```diff
@@ -143,7 +145,12 @@
 elseif ($func == "delete") {
     $locale = validate_locale($_REQUEST['locale']);
     assert(is_dir("$dyn_locales_dir/$locale"));
-    exec("rm -r " . escapeshellarg("$dyn_locales_dir/$locale"));
+    $process = new Process([
+        "rm",
+        "-r",
+        "$dyn_locales_dir/$locale",
+    ]);
+    $process->run();
 
```

Comment:
This seems to be an existing error that we don't check if the run was successful. Is it worth adding a TODO about it?

Reply:
What would be the appropriate way to handle such an error? Just log it? Or report it back to the UI?

Reply:
I would probably throw a `RuntimeException()` in this instance. Unfortunately we don't yet have a standardized logging mechanism.

Reply:
Ok, that's easy. It looks like the exception message strings should be localized, although most of them I see in the code are not. What's the policy for this?

Reply:
Either is fine. It looks like in general we haven't localized `RuntimeExceptions` probably either because we expect those to be things that the user hasn't done / can't control and rare or, more likely, we just weren't thinking about it too much.
```bash
cpeel@test:~/u-cpeel-dp $ grep -r RuntimeException * | grep -v vendor | grep -c '_('
1
cpeel@test:~/u-cpeel-dp $ grep -r RuntimeException * | grep -v vendor | grep -cv '_('
20
```